### PR TITLE
bug: Add ErrCasdoorResponse

### PR DESCRIPTION
**Pull request by hound672**
Fix: https://github.com/casdoor/casdoor-go-sdk/issues/60

### CODE REVIEW

File: casdoorsdk/util.go [hunk 1]
```diff
@@ -16,12 +16,17 @@ package casdoorsdk
...
+	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
 	"mime/multipart"
 	"strings"
 )
 
+// ErrCasdoorResponse returns while receiving invalid response from casdoor
+var ErrCasdoorResponse = errors.New("casdoor response")
```
Comment (hsluoyz) [inline casdoorsdk/util.go:28]: What's this? Is this better than the original error? I don't think so

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
It's not bug

File: casdoorsdk/util.go (hunk 1 above)
Comment (hound672) [reply casdoorsdk/util.go:28]: I was writting about it in issue: https://github.com/casdoor/casdoor-go-sdk/issues/60

Comment (hound672) [reply casdoorsdk/util.go:28]:
```
users, err := casdoorsdk.GetUsers()
if errors.Is(err, ErrCasdoorResponse) {
   // 
}```

Comment (hsluoyz) [reply casdoorsdk/util.go:28]: "casdoor response" this string is just not making sense.

**Comment by hsluoyz** [comment]
Closed as this PR doesn't resolve anything actually

**Pull request closed without merging**